src/main/host/host.rs

```diff
@@ -92,6 +92,9 @@ pub struct Host {
     random: RootedRefCell<Xoshiro256PlusPlus>,
     params: HostParameters,
 
+    // TODO: Use a Rust address type.
+    default_address: RootedRefCell<SyncSendPointer<cshadow::Address>>,
```

Comment:
I think we could use just a `RefCell` here since it only needs to be `Send`? But either is fine I think.

Reply:
Oops, right. Done.